은아/Week7_closure_1.js

```diff
@@ -0,0 +1,8 @@
+function sum(x) {
+    return function(y) {
+        return x+y;
```

Comment:
사소하지만 x+y 라고 표현하는거보단 x + y가 더 괜찮긴합니다~
```suggestion
        return x + y;
```

Reply:
요런것들은 prettier 설정을 통해 일정한 규칙으로 작성하도록 정의할 수 있습니다! -지나가다 들린 졸업생-